src/Makefile

```diff
@@ -2,6 +2,6 @@ CC=g++
 CFLAGS=-Wall -std=c++11
 
 all:
-	$(CC) $(CFLAGS) main.cpp FGame.cpp FCharacter.cpp FEngine.cpp FLevel.cpp FObject.cpp FVector.cpp FWeapon.cpp FProjectile.cpp FPowerup.cpp -lsfml-graphics -lsfml-window -lsfml-system
+	$(CC) $(CFLAGS) main.cpp FGame.cpp FCharacter.cpp FEngine.cpp FLevel.cpp FObject.cpp FVector.cpp FWeapon.cpp FProjectile.cpp FPowerup.cpp FTile.cpp -lsfml-graphics -lsfml-window -lsfml-system
 clean:
 	rm -f *.o FightR2
```

Comment:
I see there are so many files, I have dealt with this long time ago.
I have created this fancy makefile to deal with everything a bit easily, check it out:
https://github.com/dekespo/StatPP/blob/master/Makefile 

If you want something like this, I can do it for you (although bin and obj directories are missing in this repo)